src/fidesops/ops/api/v1/scope_registry.py

```diff
@@ -24,6 +24,8 @@
 )
 PRIVACY_REQUEST_REVIEW = "privacy-request:review"
 
+IDENTITY_VERIFICATION_READ = "identity-verification:read"
```

Comment:
This endpoint may require having not permissions to call in. Right now the Privacy center is calling the `create_privacy_request` endpoint with an authenticated client. AFAIK the Privacy center is unauthenticated. I'll verify that now.

Reply:
I just finished double checking and the only request to the backend is unauthenticated. I think we may need to remove the permission for this increment and circle back if it's required.

https://github.com/ethyca/fidesops/blob/main/clients/ops/privacy-center/components/RequestModal.tsx#L89-L97